modular_zubbers/code/datums/id_trim/jobs.dm

```diff
@@ -35,4 +53,4 @@
 
 /datum/id_trim/job/nanotrasen_consultant
 	minimal_wildcard_access = list()
-	template_access = list(ACCESS_CAPTAIN) //NTRep can no longer change to any other ID trim
+	template_access = list(ACCESS_CAPTAIN) //NTRep can no longer change to any other ID trim
```

Comment:
unrelated change

Reply:
Actually this whole file is unrelated changes- this upstream merge seems broken: https://github.com/Bubberstation/Bubberstation/pull/268/commits/298feaa3b8cad315c347bfc326d820f29118cbb2